src/components/QuranReader/TafsirView/TafsirView.module.scss

```diff
@@ -1,13 +1,17 @@
 @use "src/styles/constants";
 @use "src/styles/utility";
+@use "src/styles/breakpoints";
 
 .container {
-  max-width: 80%;
+  max-width: 90%;
   margin-block-start: var(--spacing-small);
   margin-block-end: var(--spacing-small);
   margin-inline-start: auto;
   margin-inline-end: auto;
   border-bottom: 1px solid var(--color-primary-medium);
+  @include breakpoints.tablet {
```

Comment:
usually when there are > 100 characters per line. The text becomes less comfortable to read. I forgot where I read this. 

But you can see how medium.com do this. They are aimin +- 80 character per line.

So we're limiting the max-width here, the number of characters per line is < 100